canonical/ubuntu.com
# [WD-8137] change link in Ubuntu Pro legal page

## Done

- Direct `View the Ubuntu Pro service description` link on /legal/ubuntu-pro page to /legal/ubuntu-pro-description
- Redirect /legal/ubuntu-pro-description URL to new PDF asset link provided in [copy doc](https://docs.google.com/document/d/1p-xlkejIG5ZZhOfGrYnq6rmIpTpaX46yW4HArDtPX1s/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /legal/ubuntu-pro page and check the `View the Ubuntu Pro service description` link

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8137

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
